addons/website_sale/static/src/xml/website_sale_image_viewer.xml

```diff
@@ -24,7 +24,10 @@
                                 <t t-foreach="images" t-as="image" t-key="image.thumbnailSrc">
                                     <li t-attf-class="align-top position-relative px-1 pb-1 {{image === selectedImage ? 'active' : ''}}" t-on-click="() => this.selectedImage = image">
                                         <div>
-                                            <img t-att-src="image.thumbnailSrc" t-attf-class="img o_wsale_image_viewer_thumbnail {{image === selectedImage ? 'active' : ''}}" t-att-alt="props.title" loading="lazy"/>
+                                            <img t-att-src="image.thumbnailSrc"
+                                                t-attf-class="img o_wsale_image_viewer_thumbnail object-fit-cover {{image === selectedImage ? 'active' : ''}}"
```

Comment:
Let's use `contain` 😉

|  |
|--------|
| ![image](https://github.com/user-attachments/assets/594087dc-c8f1-41ce-888b-64924d1ea425) | 


```suggestion
                                                t-attf-class="img o_wsale_image_viewer_thumbnail bg-body object-fit-contain {{image === selectedImage ? 'active' : ''}}"
```

Reply:
.. and don't forget the mobile adaptation we discussed about ;)